x-pack/plugins/cloud_security_posture/public/pages/findings/findings_container.tsx

```diff
@@ -40,8 +37,23 @@ export const FindingsContainer = ({ dataView }: { dataView: DataView }) => {
         {...findingsQuery}
         {...findingsResult}
       />
-      <EuiSpacer />
-      <FindingsTable setQuery={setUrlQuery} {...findingsQuery} {...findingsResult} />
+      <div
+        css={css`
+          padding: 24px;
```

Comment:
We need to start using sizes from the eui theme, lets use `size.l` instead of a fixed value here